File: src/main/java/it/contrader/dto/QuestionDTO.java
package it.contrader.dto;

import it.contrader.model.Company;
import it.contrader.model.Recruiter;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class QuestionDTO {
	private long id;
	private String question;
	private String topic;
	private String sector;
	private Recruiter recruiter;
	private Company company;
}

File: src/dto/questiondto.ts
import { RecruiterDTO } from './recruiterdto';
import { CompanyDTO } from './companydto';

export class QuestionDTO {
    id: number;
    question: string;
    topic: string;
    recruiterId: number;
    recruiterName: string;
    companyName: string;
    sector: string;
}
